lib/elixir_sonic_client.ex

```diff
@@ -25,17 +30,34 @@ defmodule ElixirSonicClient do
   end
 
   @doc """
-  Start Connection with Sonic Server.
+  Stop connection with Sonic server
+  """
+  def stop(conn) do
+    with(
+      :ok <- TcpConnection.send(conn, "QUIT"),
+      {:ok, _msg} <- TcpConnection.recv(conn)
+    ) do
+      TcpConnection.close(conn)
+    end
+  end
 
-  ## Examples
+  @doc """
+  Send PING message to Sonic server
 
-      iex> {:ok, conn} = ElixirSonicClient.start(127.0.0.1, 1491, "search")
+  ## Examples
       iex> {:ok, conn} = ElixirSonicClient.ping(conn)
-      PONG
-
+      {:ok, "PONG"}
   """
   def ping(conn) do
     TcpConnection.send(conn, "PING")
     TcpConnection.recv(conn)
   end
+
+  defdelegate push(conn, collection, object, term), to: Ingest
```

Comment:
I'm wondering if it wouldn't be clearer to not delegate those functions and just use `Ingest.push(conn, collection, object, term)`.  Do you have any opinion about that?

Reply:
Nah, I do not have much of an opinion, no. Only doing what I've seen in https://github.com/imerkle/sonix The body of this function would not by much more that a call to `Ingest.push` with exactly the same arguments, right? It's a reexport of sorts, so to speak.

No problem at all with using a regular function call, though. Will change this.